README.md

```diff
@@ -1,106 +1,21 @@
-# Storybook Addon Kit
+# Storybook Addon Angular Ivy
 
-Simplify the creation of Storybook addons
+This addon configures [Storybook for Angular](https://storybook.js.org/docs/angular/get-started/introduction) to use the [Angular Ivy compiler](https://angular.io/guide/ivy) instead of the just-in-time compiler that Storybook uses by default.
```

Comment:
Not entirely correct. Ivy still uses just-in-time compilation in our case because the support for AoT is quite difficult to achieve.

There is Ivy (new) and ViewEngine (old) - both are able to either use just-in-time or ahead-of-time compilation

```
# https://angular.io/guide/ivy
Ivy is the code name for Angular's next-generation compilation and rendering pipeline. With the version 9 release of Angular, the new compiler and runtime instructions are used by default instead of the older compiler and runtime, known as View Engine.
```